src/components/ListingCard.tsx

```diff
@@ -1,47 +1,46 @@
-import { Badge } from "@/components/Badge";
-import { findListingById } from "@/services/listing";
 import { formatPrice } from "@/utils/format";
-import Image from "next/image";
 import Link from "next/link";
+import { findListingById } from "../services/listing";
+import PriceBadge from "./PriceBadge";
 
 type Props = {
   /**Listing型 表示に必要なリレーション先のテーブルをインクルード済み */
   listing: Awaited<ReturnType<typeof findListingById>>;
 };
 
 /**
- * 商品のカード
- * @param param0.listing 商品
- * @returns 商品のカード
+ * 商品カードを表示するコンポーネント
+ * @param {Props} props - コンポーネントに渡されるプロパティ
+ * @param {ReturnType<typeof findListingById>} props.listing - 表示に必要なリレーション先のテーブルをインクルード済みのListing型のオブジェクト
+ * @returns {JSX.Element} 商品カードのJSX要素を返します。
+ * @example
+ * <ListingCard listing={sampleListing} />
  */
 export function ListingCard({ listing }: Props) {
-  // 製品が作成されて7日以内
-  const isNew =
-    Date.now() - new Date(listing.createdAt!).getTime() <
-    1000 * 60 * 60 * 24 * 7;
+  const formattedPrice =
+    listing.price != null ? formatPrice(listing.price) : "N/A";
 
   return (
-    <Link
-      href={"/listings/" + listing.id}
-      className="card w-full bg-base-100 transition-shadow hover:shadow-xl"
-    >
-      <figure>
-        <Image
-          src={listing.images[0].image.imageURL}
-          alt={listing.productName!}
-          width={800}
-          height={400}
-          className="h-48 object-cover"
-        />
-      </figure>
-      <div className="card-body">
-        <h2 className="card-title">
-          {listing.productName}
-          {isNew && <Badge variant="secondary">New</Badge>}
-        </h2>
-        <p>{listing.description}</p>
-        <Badge>{formatPrice(listing.price!)}</Badge>
+    <div className="">
+      <div className="relative flex items-center justify-center rounded-lg bg-gray-300">
+        <Link
+          href={"/listings/" + listing.id}
+          className="card w-full bg-base-100 transition-shadow hover:shadow-xl"
+        >
+          <div
+            className="h-32 w-32 cursor-pointer rounded-lg object-cover sm:h-48 sm:w-48"
+            style={{
```

Comment:
Tailwind CSSとNext/Imageを使ってほしいです。

```suggestion
    <Image src={listing.images[0].image.imageURL} fill objectFit="cover" >
```
このようにすれば相対表示でwidthとheightを指定せずに画像を使えたと思います。

Reply:
@bmthd 

お疲れ様です！
修正しました。
お願いします。